api/task_processor/processor.py

```diff
@@ -28,6 +28,8 @@ def run_tasks(num_tasks: int = 1) -> typing.List[TaskRun]:
         executed_tasks = []
         task_runs = []
 
+        logger.debug("Processing %d tasks", len(list(tasks)))
```

Comment:
Wouldn't this work better?

```suggestion
        logger.debug("Processing %d tasks", len(tasks))
```

Reply:
I'm not sure I can call `len()` on a `RawQuerySet` class - you certainly can't on a `QuerySet` class. I tried to use `.count()` but that doesn't exist on the `RawQuerySet` class.